wideq.py

```diff
@@ -806,6 +818,31 @@ def set_fahrenheit(self, f):
 
         self.set_celsius(self.f2c[f])
 
+    def set_zones(self, zones):
+        """Set the device's zones to on/off.
+        zones arg is a list of the format below
+        zone_no is a number indexed from 1 typed as string
+        enabled is a bool typed as string
+        isOpen is a bool typed as string
+        [{'No':zone_no, 'Cfg':enabled, 'State':isOpen},]
+        """
+        #ensure at least 1 zone is enabled. Can't turn all zones off
+        on_count = sum(int(zone['State']) for zone in zones)
+        if(on_count > 0):
+            zone_cmd = '/'.join(zone['No']+'_'+zone['State'] for zone in zones if zone['Cfg'] == '1')
+            self._set_control('DuctZone', zone_cmd)
+
+    def get_zones(self):
+        """Gets the status of the zones, including whether a zone is configured.
+        Result is a list of dicts with the same format as set_zones()
+        """
+        return self._get_config('DuctZone')
+
+    def set_fan_speed(self, speed):
+        """Sets the fan speed according to the WindStreng operation"""
```

Comment:
WindStreng -> WindStrength

Maybe this docstring should also clarify that `speed` is a value of the `ACFanSpeed` enum?